web/src/subscriber_api.ts

```diff
@@ -1,19 +1,24 @@
 import * as channel from "./channel";
 import * as people from "./people";
-
+import type { StreamSubscription } from "./sub_store";
 /*
     This module simply encapsulates our legacy API for subscribing
     or unsubscribing users from streams. Callers don't need to
     know the strange names of "subscriptions" and "principals",
     nor how to JSON.stringify things, nor the URL scheme.
 */
 
-export function add_user_ids_to_stream(user_ids, sub, success, failure) {
+//declaring type for the function parameters
+type ids=number[];
+type succ_type=()=>void;
+type fail_type=(xhr:JQuery.jqXHR<unknown>)=>void;
```

Comment:
These are very simple types; just write them inline.

Reply:
Sure @andersk . I will refract it asap.thanks for the review